sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -29,6 +31,26 @@ export type AuthorizationRule = {
     secondaryKey?: string;
 };
 
+// @public
+export interface Closeable {
+    close(): Promise<void>;
+}
+
+// @public
+export type ContextType<LockModeT> = LockModeT extends "peekLock" ? ContextWithSettlement : LockModeT extends "receiveAndDelete" ? {} : never;
+
+// @public
+export interface ContextWithSettlement {
+    abandon(message: ReceivedMessage, propertiesToModify?: {
```

Comment:
If the context always accompanies a message then why should the settlement methods take the message as an argument?
If they do indeed take the message as an argument, then do we really need a new "context" for each message?